src/components/Energiatodistus/Energiatodistukset.svelte

```diff
@@ -70,35 +104,49 @@
     <div slot="content">
       {#if R.isEmpty(energiatodistukset)}
         <p class="mb-10">
-          Energiatodistuksia ei löydy annetuilla hakuehdoilla
-          tai sinulle ei ole yhtään energiatodistusta.
+          Energiatodistuksia ei löydy annetuilla hakuehdoilla tai sinulle ei ole
+          yhtään energiatodistusta.
         </p>
       {:else}
         <div class="mb-10">
           <table>
             <thead>
-            <tr>
-              <th>Tila</th><th>Tunnus</th><th>ETL</th>
-              <th>Versio</th><th>Voimassa</th>
-              <th>Rakennuksen nimi</th><th>Osoite</th>
-              <th>Laatija</th>
-              <th>Toiminnot</th>
-            </tr>
+              <tr>
+                <th>Tila</th>
+                <th>Tunnus</th>
+                <th>ETL</th>
+                <th>Versio</th>
+                <th>Voimassa</th>
+                <th>Rakennuksen nimi</th>
+                <th>Osoite</th>
+                <th>Laatija</th>
+                <th>Toiminnot</th>
+              </tr>
             </thead>
             <tbody>
               {#each energiatodistukset as energiatodistus}
-                <tr class="cursor-pointer"
-                    on:click={toETView(energiatodistus.versio, energiatodistus.id)}>
+                <tr
+                  class="cursor-pointer"
+                  on:click={toETView(energiatodistus.versio, energiatodistus.id)}>
                   <td>Luonnos</td>
                   <td>{energiatodistus.id}</td>
                   <td>C</td>
                   <td>{energiatodistus.versio}</td>
                   <td>13.3.2030</td>
                   <td>{orEmpty(energiatodistus.perustiedot.nimi)}</td>
-                  <td>{orEmpty(energiatodistus.perustiedot['katuosoite-fi'])}</td>
+                  <td>
+                    {orEmpty(energiatodistus.perustiedot['katuosoite-fi'])}
+                  </td>
                   <td>{orEmpty(energiatodistus['laatija-fullname'])}</td>
                   <td>
-                    <span class="material-icons">delete</span>
+                    <span
+                      class="material-icons"
+                      on:click={e => {
+                        e.stopPropagation();
```

Comment:
Tän voi myös kirjoittaa näin 
`on:click|stopPropagation={_ => deleteEnergiatodistus(energiatodistus.versio, energiatodistus.id)}`

Tietty jos nyt nappaa developista rebasen, niin siellä löytyy Confirm-komponentti jolla on mahdollista saada lisävarmistus tuohon.